app/assets/stylesheets/views/_search.scss

```diff
@@ -133,6 +133,24 @@ main.search {
         float: left;
         width: $two-thirds;
       }
+
+      mark {
+        background-color: rgba(255, 255, 140, 0.5);
```

Comment:
@quis The large text (>=18px) passes the AA checks, so that is OK.

We are highlighting the titles because we are using synonyms and want people to see that their term may have matched a synonym. We also want it to be visible to those using high contrast screens (the same reason we use the focus styles as well).